src/git-commit-push-via-github-api.ts

```diff
@@ -150,7 +150,7 @@ export const gitCommitPush = (options: GitCommitPushOptions) => {
     }
     const gitHub = new GitHubApi();
     if (token) {
-        gitHub.authenticate({
+        gitHub.auth({
```

Comment:
https://github.com/octokit/rest.js/#authentication
This deprecation message means that use `auth` option of Constructor like following.

```
const gitHub = new GitHubApi({ 
  auth: "token secret"
});
```